sdk/ruby/CHANGELOG.md

```diff
@@ -0,0 +1,5 @@
+## [Unreleased]
+
+## [0.1.0] - 2025-01-31
```

Comment:
I think you said you were going with 1.0.0-beta.1?

Reply:
Right. The Changelog file was auto-generated by my tooling. I'll remove it for now. Don't need the overhead until this is stable IMO.